src/person.rb

```diff
@@ -20,6 +20,8 @@ def correct_name
     @name
   end
 
+  has_many :rental
```

Comment:
- [ ] Please refer to this [lesson](`https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship`) to understand how this feature should be implemented.